### PR TITLE
Currency Component: Support other currencies - Add suffix / prefix options

### PR DESCRIPTION
**Pull request by abarrak**
This improves the currency component by exposing `currency`, `prefix`, `suffix` attributes via the builder.

Additionally, [browser-supported](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString) ISO-4217 currencies are maid available as select list.

currencies 
suffix-prefix

### CODE REVIEW

**Comment by travist** [comment]
Sweet!  Thank you @abarrak!!!

File: src/components/currency/editForm/Currency.edit.display.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+export default [
+  {
+    type: 'textfield',
+    input: true,
+    weight: 310,
+    key: 'prefix',
```
Comment (travist) [inline src/components/currency/editForm/Currency.edit.display.js]: We already have a ```prefix``` and ```suffix``` for the text field components. We should either use different names for these or hide the ones from the display tab.

Comment (abarrak) [reply src/components/currency/editForm/Currency.edit.display.js]: @travist could you clarify more?

As far as I can see ‘Currency’ inherits from ‘Number’, and has differed suffix/prefix defined [here](
https://github.com/formio/formio.js/pull/866/files#diff-46b3380c5b96fc2e752f50913ef0c35cR40) ?

Comment (travist) [reply src/components/currency/editForm/Currency.edit.display.js]: Yeah OK, I think this is good.

**Pull request merged**